<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordWidgetClass</class>
 <widget class="QWidget" name="PasswordWidgetClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>9</pointsize>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="oldLabel">
     <property name="text">
      <string>旧密码</string>
     </property>
     <property name="buddy">
      <cstring>oldLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="newLabel">
     <property name="text">
      <string>新密码</string>
     </property>
     <property name="buddy">
      <cstring>newLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="newLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="confirmLabel">
     <property name="text">
      <string>确认新密码</string>
     </property>
     <property name="buddy">
      <cstring>confirmLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="confirmLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submitPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="oldLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordWidgetClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oldLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PasswordWidgetClass</receiver>
   <slot>checkInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PasswordWidgetClass</receiver>
   <slot>checkInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PasswordWidgetClass</receiver>
   <slot>checkInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordWidgetClass</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkInput()</slot>
  <slot>submit()</slot>
 </slots>
</ui>
